src/components/Auth/AuthForms/LoginForm.jsx

```diff
@@ -0,0 +1,64 @@
+import { FormControl, FormHelperText, TextField } from "@material-ui/core";
+import clsx from "clsx";
+import React from "react";
+
+import { validateEmail } from "../../../utils/email";
+import formStyle from "../../../utils/formStyles";
+
+import "./AuthForm.css";
+
+const LoginForm = ({
+  message,
+  loginDetails,
+  setLoginDetails,
+  errors,
+  loading,
+}) => {
+  const classes = formStyle();
+  const handleLoginDetails = prop => event => {
+    setLoginDetails({
+      ...loginDetails,
+      [prop]: {
+        ...loginDetails[prop],
+        value: event.target.value,
+        isValid: validateEmail(loginDetails.email.value),
+      },
+    });
+  };
+  return (
+    <>
+      <FormControl fullWidth>
+        <FormHelperText className={clsx(classes.helperText)}>
+          {message}
+        </FormHelperText>
+        <TextField
```

Comment:
Both the `TextField`s are good candidates for extraction to eg `EmailTextField` and `PasswordTextField`. Constants contained within. The whole `...props` passed from parent component (shorter = easier to read), whereas the child components (`EmailTextField` & `PasswordTextField`) would destructure as usual.